modules/t/MySQLImportHomologies.t

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Utils::Test qw(standaloneJob);
+use Bio::EnsEMBL::Hive::DBSQL::DBConnection;
+use Bio::EnsEMBL::Test::MultiTestDB;
+
+use Test::Most;
+
+BEGIN {
+    # check module can be seen and compiled
+    use_ok('Bio::EnsEMBL::Compara::RunnableDB::Flatfiles::MySQLImportHomologies');
+}
+
+##################################################################################
+#                              Test homologies                                   #
+##################################################################################
+subtest "Test Bio::EnsEMBL::Compara::RunnableDB::Flatfiles::MySQLImportHomologies", sub {
+    # Load test DB #
+    my $multi_db = Bio::EnsEMBL::Test::MultiTestDB->new('mysqlimport_test');
+    my $dba = $multi_db->get_DBAdaptor('compara');
+    my $dbc = Bio::EnsEMBL::Hive::DBSQL::DBConnection->new(-dbconn => $dba->dbc);
+    my $compara_db = $dbc->url;
+    $dbc->do("TRUNCATE TABLE homology");
+
+    # find absolute path to the test input
+    # important for travis-ci
+    use Cwd 'abs_path';
+    my $test_flatfile_dir = abs_path($0);
+    $test_flatfile_dir =~ s!MySQLImportHomologies\.t!homology_flatfiles/mysqlimport_test!;
+
+    # run in standalone
+    standaloneJob(
+        'Bio::EnsEMBL::Compara::RunnableDB::Flatfiles::MySQLImportHomologies',
+        {
+            'compara_db'   => $compara_db,
+            'homology_flatfile' => "$test_flatfile_dir/homologies.tsv",
+            'attrib_files' => [
+                "$test_flatfile_dir/wga.tsv",
+                "$test_flatfile_dir/goc.tsv",
+                "$test_flatfile_dir/high_conf.tsv",
+            ],
+            'homology_id_start' => 11,
+        },
+    );
+
+    # check main homology data loading
+    my $hom_sth = $dba->dbc->prepare("SELECT 
```

Comment:
Comment: you could have uses `selectall_arrayref` or the `sql_helper->execute` here and below, then you don't need to call `execute` and `fetchall_arrayref`